pyproject.toml

```diff
@@ -106,7 +106,7 @@ test = [
     "pytest-doctestplus>=1.4.0",
     "pytest-astropy-header>=0.2.1",
     "pytest-astropy>=0.10.0",
-    "pytest-xdist>=2.5.0",
+    "pytest-xdist>=3.0.2",
```

Comment:
the reason I'm bumping this package is that version 2.5.0 implicitly requires the `py` package, which used to be distributed with `pytest` (originally, `py.test`), but isn't part of pytest 8.0.0.
We are not capturing this problem in `oldestdeps` CI runs because it doesn't use `pytest-xdist` and runs all tests sequentially instead.